client/src/components/AddBookModal/AddBookModal.tsx

```diff
@@ -78,16 +88,21 @@ const AddBookModal: React.FC<AddBookModalProps> = ({closeModalFunc, addBookFunct
   } else if(modalContent === 3) {
     content = 
       <div>
-        <p>Can you lend this book?</p>
-        <div className="checkboxDiv">
-          <input type="checkbox" value='yes' name='availableToBorrow' onChange={(e) => handleChange(e)} checked={userInput.availableToBorrow ? true: false}/>
+        <h2>Can you lend this book?</h2>
+        <div className='addbook-modal-grand-wrapper__pop-out-container__checkbox-div'>
+          <input 
+            type="checkbox"
+            checked={userInput.availableToBorrow}
+            name='availableToBorrow' 
+            onChange={(e) => handleChange(e)} 
```

Comment:
Also, just as an FYI, names for html elements like checkboxes should follow html naming conventions, so in this case it would be `name='available-to-borrow'`